src/lozad.js

```diff
@@ -16,51 +48,19 @@
                 this.observer.disconnect();
             }
             this.observer = new IntersectionObserver(onIntersection.bind(this), this.config);
-            var elements = getElements.call(this);
-            for (var i = 0; i < elements.length; i++) {
-                if (!isLoaded(elements[i])) {
-                    this.observer.observe(elements[i]);
+            const elements = getElements.call(this);
+            elements.forEach(function (element) {
```

Comment:
Why not just do this?

```js
elements.filter(!isLoaded).map(el => this.observer.observe(el));
```

Reply:
Thanks for your comment :)
First, we can't just call `this.getElements()` because `getElements` is just a 'private' function inside this closure. And the return value of `getElements` is NodeList, so we can't use `filter` or `map` on it.
And the reason for not using the concise arrow function is because "grunt-contrib-uglify" hasn't support arrow function in its release yet :/   https://github.com/gruntjs/grunt-contrib-uglify/issues/393

Reply:
The NodeList can be converted to an array with Array.from: `Array.from(getElements.call(this)).filter(!isLoaded).map()`

Maybe that could be called inside getElements so it always returns an array 🤓 

The uglify problem can likely be fixed by changing [uglify-js](https://www.npmjs.com/package/uglify-js) to [uglify-es](https://www.npmjs.com/package/uglify-es).

Reply:
I think that may be overkilled, as this PR is just a simple refactor as I mentioned in PR title to make it bit more readable. 
There will probably has more large-scale updates to the lib in the future, so I don't want to do too much unnecessary things now.